server/statshouse/statshouse-client.cpp

```diff
@@ -1,164 +1,126 @@
+  transport.metric("kphp_workers_general_processes").tag(cluster_name).tag("ready_for_accept").write_value(general_worker_group.ready_for_accept_workers);
+
+  const auto job_worker_group = vk::singleton<ServerStats>::get().collect_workers_stat(WorkerType::job_worker);
+  transport.metric("kphp_workers_job_processes").tag(cluster_name).tag("working").write_value(job_worker_group.running_workers);
+  transport.metric("kphp_workers_job_processes").tag(cluster_name).tag("working_but_waiting").write_value(job_worker_group.waiting_workers);
+
+  transport.metric("kphp_server_workers").tag(cluster_name).tag("started").write_value(workers_stats.tot_workers_started);
+  transport.metric("kphp_server_workers").tag(cluster_name).tag("dead").write_value(workers_stats.tot_workers_dead);
+  transport.metric("kphp_server_workers").tag(cluster_name).tag("strange_dead").write_value(workers_stats.tot_workers_strange_dead);
+  transport.metric("kphp_server_workers").tag(cluster_name).tag("killed").write_value(workers_stats.workers_killed);
+  transport.metric("kphp_server_workers").tag(cluster_name).tag("hung").write_value(workers_stats.workers_hung);
+  transport.metric("kphp_server_workers").tag(cluster_name).tag("terminated").write_value(workers_stats.workers_terminated);
+  transport.metric("kphp_server_workers").tag(cluster_name).tag("failed").write_value(workers_stats.workers_failed);
+
+  transport.metric("kphp_cpu_usage").tag(cluster_name).tag("stime").write_value(cpu_s_usage);
+  transport.metric("kphp_cpu_usage").tag(cluster_name).tag("utime").write_value(cpu_u_usage);
+
+  auto total_workers_json_count = vk::singleton<ServerStats>::get().collect_json_count_stat();
+  uint64_t master_json_logs_count = vk::singleton<JsonLogger>::get().get_json_logs_count();
+  transport.metric("kphp_server_total_json_logs_count").tag(cluster_name).write_value(std::get<0>(total_workers_json_count) + master_json_logs_count);
+  transport.metric("kphp_server_total_json_traces_count").tag(cluster_name).write_value(std::get<1>(total_workers_json_count));
+
+  transport.metric("kphp_instance_cache_memory").tag(cluster_name).tag("limit").write_value(memory_stats.memory_limit);
+  transport.metric("kphp_instance_cache_memory").tag(cluster_name).tag("used").write_value(memory_stats.memory_used);
+  transport.metric("kphp_instance_cache_memory").tag(cluster_name).tag("real_used").write_value(memory_stats.real_memory_used);
+  transport.metric("kphp_instance_cache_memory").tag(cluster_name).tag("defragmentation_calls").write_value(memory_stats.defragmentation_calls);
+  transport.metric("kphp_instance_cache_memory").tag(cluster_name).tag("huge_memory_pieces").write_value(memory_stats.huge_memory_pieces);
+  transport.metric("kphp_instance_cache_memory").tag(cluster_name).tag("small_memory_pieces").write_value(memory_stats.small_memory_pieces);
```

Comment:
It's strange to combine bytes and counts in one metric, they have different units. Let's divide it